### ISSUE TITLE
Interest in working with OSS devs to make a user-facing Thelio IO dashboard app

### ISSUE THREAD

**Issue by NoraCodes** [system76/thelio-io#1]
Hi! I'm a desktop Linux app developer (and Thelio early adopter) and I'd be really interested in working with System76 to develop a desktop app, including both a command line and GUI component, that end users could use to tweak parameters of their I/O boards, mostly around the fan profiles.

Is this something System76 would be interested in assisting with?

**Comment by jackpot51** [comment]
Yes, we may be interested. The fan curves are not defined in the thelio-io firmware, but instead in software here: https://github.com/pop-os/system76-power

https://github.com/pop-os/system76-power/blob/master/src/fan.rs#L209
